addons/project/models/project.py

```diff
@@ -760,6 +760,12 @@ def _mail_get_message_subtypes(self):
     def action_project_task_burndown_chart_report(self):
         action = self.env['ir.actions.act_window']._for_xml_id('project.action_project_task_burndown_chart_report')
         action['display_name'] = _("%(name)s's Burndown Chart", name=self.name)
+        context = action['context'].replace('active_id', str(self.id))
+        context = ast.literal_eval(context)
+        context.update({
+            'stage_seq': {stage.id: {'sequence': stage.sequence, 'name': stage.name} for stage in self.type_ids}
+            })
```

Comment:
```suggestion
        context.update({
            'stage_name_and_sequence_per_id': {
                stage.id: {
                    'sequence': stage.sequence,
                    'name': stage.name
                } for stage in self.type_ids
            }
        })
```
- Close the brackets you opened at same indent.
- Name the dict more explicitly.
- Unwrap it totally.